usaspending_api/awards/models.py

```diff
@@ -211,13 +211,13 @@ class Award(DataSourceTrackedModel):
     certified_date = models.DateField(blank=True, null=True, help_text="The date this record was certified")
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True,
                                        help_text="The date this record was created in the API")
-    update_date = models.DateTimeField(auto_now=True, null=True,
+    update_date = models.DateTimeField(auto_now=True, null=True, db_index=True,
```

Comment:
I was about to leave a comment that this index might not be necessary. But it has been used 227 times since the postgres stats were last reset.

However, in the DB it is sorted DESC. We might need to use the Index object in the meta class method to add the existing ordering

e.g. `Index(fields=['-update_date'])`

Reply:
Your call if you want to make it ordered DESC or simply drop since it is hardly used